setuptools/config/setupcfg.py

```diff
@@ -43,19 +43,25 @@
     from distutils.dist import DistributionMetadata
 
     from setuptools.dist import Distribution
+    from typing_extensions import TypeAlias
 
-SingleCommandOptions = Dict["str", Tuple["str", Any]]
-"""Dict that associate the name of the options of a particular command to a
-tuple. The first element of the tuple indicates the origin of the option value
-(e.g. the name of the configuration file where it was read from),
-while the second element of the tuple is the option value itself
-"""
-AllCommandOptions = Dict["str", SingleCommandOptions]  # cmd name => its options
+    SingleCommandOptions: TypeAlias = Dict[str, Tuple[str, Any]]
+    """Dict that associate the name of the options of a particular command to a
+    tuple. The first element of the tuple indicates the origin of the option value
+    (e.g. the name of the configuration file where it was read from),
+    while the second element of the tuple is the option value itself
+    """
+    AllCommandOptions: TypeAlias = Dict[str, SingleCommandOptions]
+    """cmd name => its options"""
+
+else:
+    SingleCommandOptions = Dict[str, Tuple[str, Any]]
+    AllCommandOptions = Dict[str, SingleCommandOptions]
```

Comment:
What is the big advantage here of keeping two copies in the `if/else`? Isn't it better to just keep everything outside of the if as in the original?